lib/stormpath-sdk/resource/collection.rb

```diff
@@ -51,6 +51,10 @@ def limit limit
     self
   end
 
+  def size
+    current_page.items.size
```

Comment:
You are right, but in the SDK that collection resource is represented by a [`CollectionPage`](https://github.com/stormpath/stormpath-sdk-ruby/blob/master/lib/stormpath-sdk/resource/collection.rb#L86) object. So that means that we should add the `size` property to the `CollectionPage` class and remove it from the `Collection` class.